packages/config/src/bridges/symbiosis.ts

```diff
@@ -0,0 +1,167 @@
+import { ProjectId, UnixTime } from '@l2beat/shared-pure'
+
+import { ProjectDiscovery } from '../discovery/ProjectDiscovery'
+import { NUGGETS } from '../layer2s'
+import { RISK_VIEW } from './common'
+import { Bridge } from './types'
+
+const discovery = new ProjectDiscovery('symbiosis')
+
+export const symbiosis: Bridge = {
+  type: 'bridge',
+  id: ProjectId('symbiosis'),
+  display: {
+    name: 'Symbiosis',
+    slug: 'symbiosis',
+    category: 'Hybrid',
+    description:
+      'Symbiosis is a cross-chain AMM DEX that pools together liquidity from different networks, whether they use EVM technology or not.',
```

Comment:
rewrite the desciption, make it sound more professional, also include some info about how it is validated

Reply:
Hi! 
Thank for the feedback.


`make it sound more professional`
Could you please provide a link to any example that sounds really professional?


`include some info about how it is validated`
I'm afraid I don't get it, what you mean by validated?   It works this way right now, so everyone can go to the app and validate it by his own hands.

Reply:
i'll take care of it :)